aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,8 +1,315 @@
+
+=============  ===============================================================================================================================  
+Module Name	   Description    
+=============  ===============================================================================================================================  
+gulp	       The gulp streaming build system. For more information, see gulp.
+rimraf	       A Node deletion module. For more information, see rimraf.
+gulp-concat	   A module that will concatenate files based on your operating systems newline character. For more information, see gulp-concat.
+gulp-cssmin	   A module that will minify CSS files. For more information see gulp-cssmin.
+gulp-uglify	   A module that minifies files using the UglifyJS toolkit. For more information, see gulp-uglify. 
+=============  =============================================================================================================================== 
+
+Once modules are registered from *gulpfile.js*, the tasks are specified. Visual Studio 2015 registers six tasks based on the following code contained in *gulpfile.js*:
+
+.. code-block:: javascript
+	:emphasize-lines: 1,5,9,11,18,25
+
+	gulp.task("clean:js", function (cb) {
+		rimraf(paths.concatJsDest, cb);
+	});
+
+	gulp.task("clean:css", function (cb) {
+		rimraf(paths.concatCssDest, cb);
+	});
+
+	gulp.task("clean", ["clean:js", "clean:css"]);
+
+	gulp.task("min:js", function () {
+		gulp.src([paths.js, "!" + paths.minJs], { base: "." })
+			.pipe(concat(paths.concatJsDest))
+			.pipe(uglify())
+			.pipe(gulp.dest("."));
+	});
+
+	gulp.task("min:css", function () {
+		gulp.src([paths.css, "!" + paths.minCss])
+			.pipe(concat(paths.concatCssDest))
+			.pipe(cssmin())
+			.pipe(gulp.dest("."));
+	});
+
+	gulp.task("min", ["min:js", "min:css"]);
+
+The following table gives an explanation of the tasks specified in the code above:
+
+=============  ===============================================================================================================================  
+Task Name	   Description    
+=============  ===============================================================================================================================  
+clean:js	   A task that uses the rimraf Node deletion module to remove unneeded text from *.js* files.
+clean:css	   A task that uses the rimraf Node deletion module to remove unneeded text from *.css* files.
+clean	       A task that calls both the ``clean:js`` and ``clean:css`` tasks.
+min:js	       A task that matches files based on the specified JavaScript file path, concatenates the files into a destination location, removes unneeded characters using the ``uglify()`` function, and re-writes the file to the destination location.
```

Comment:
Could just say: A task that minifies and concatenates JS files. (Same comment for min:css)